.github/scripts/toolchain/execute-gcc-tests.sh

```diff
@@ -13,6 +13,8 @@ HOST_BOARD=${5:-}
 GCC_BUILD_PATH=$BUILD_PATH/gcc
 TEST_RESULTS_PATH=$ARTIFACT_PATH/gcc-tests-$TAG
 
+PATH="$TOOLCHAIN_PATH/bin:$PATH"
```

Comment:
are you sure this change is not enough?
https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/commit/bb9c477a26a087e33a2fe5b0cf20f45bc34b823e

Reply:
Let me try once again though I doubt it. That change is about host/build binaries used for detection of host/build compiler features during `configure`. This change is about what `as` executable will the resulting GCC binary use for executing during testing.

Reply:
No, it's not enought. Without this change the following errors occur:
```
Executing on host: /root/mingw-woarm64-build/build-aarch64-w64-mingw32-msvcrt/gcc/gcc/xgcc -B/root/mingw-woarm64-build/build-aarch64-w64-mingw32-msvcrt/gcc/gcc/    -fdiagnostics-plain-output    -O1  -w -c -o pr106433.o /root/mingw-woarm64-build/code/gcc-master/gcc/testsuite/gcc.c-torture/compile/pr106433.c    (timeout = 300)
spawn -ignore SIGHUP /root/mingw-woarm64-build/build-aarch64-w64-mingw32-msvcrt/gcc/gcc/xgcc -B/root/mingw-woarm64-build/build-aarch64-w64-mingw32-msvcrt/gcc/gcc/ -fdiagnostics-plain-output -O1 -w -c -o pr106433.o /root/mingw-woarm64-build/code/gcc-master/gcc/testsuite/gcc.c-torture/compile/pr106433.c
/tmp/ccu7EAi0.s: Assembler messages:
/tmp/ccu7EAi0.s:43: Error: unknown pseudo-op: `.variant_pcs'
/tmp/ccu7EAi0.s:92: Error: unknown pseudo-op: `.variant_pcs'
/tmp/ccu7EAi0.s:152: Error: unknown pseudo-op: `.variant_pcs'
/tmp/ccu7EAi0.s:201: Error: unknown pseudo-op: `.variant_pcs'
compiler exited with status 1
```

Reply:
do you think PATH can be changed in config.sh?

Reply:
Only if config.sh would accept parameter/environment variable denoting that for this case the path should be constructed for testing. E.g.,

```shell
if [[ -n "$MSYSTEM" ]]; then
    export MSYS=winsymlinks
  	PATH="$TOOLCHAIN_PATH/bin:$TOOLCHAIN_PATH/lib/gcc/$TARGET/15.0.0:/opt/bin:$PATH"

else
    if [[
 "$TEST_EXECUTE" = 1 ]]
    	PATH="$TOOLCHAIN_PATH/bin:$PATH"
    else

    	PATH="$PATH:$TOOLCHAIN_PATH/bin"
    fi
fi
if [[ "$CCACHE" = 1 && "$TEST_EXECUTE" != 1 ]]; then
    PATH=$CCACHE_LIB_DIR:$TOOLCHAIN_CCACHE_LIB_DIR:$PATH
    export CCACHE_DIR=$CCACHE_DIR_PATH
fi
```

Would you prefer this?

Reply:
I prefer the original version.